src/components/DemoCard/DemoCard.svelte

```diff
@@ -39,25 +53,90 @@
     }
   };
 
+  const onUpdate = async () => {
+    const response = await fetch(`/api/users/${$userIdStore}/demos/${uuid}`, {
+      method: 'PATCH',
+      body: JSON.stringify({ name })
+    });
+    editMode = false;
+
+    if (response.status === 500) {
+      $showErrorModalStore = true;
+    } else {
+      title = name;
+    }
+  };
```

Comment:
In the feature, custom event dispatching would be a better approach than using global variable state (`editMode`, `showErrorModalStore`).

Reply:
Please use the internal fetch wrapper to do this. Checking for response code at each fetch location will make things very verbose.